### DISCUSSION TITLE
全程使用 Docker 为本程序开启HTTPS不在需要在config.json配置需要绑定的公网域名参数了吗？

### DISCUSSION THREAD

**Discussion by limboys** [GaiZhenbiao/ChuanhuChatGPT#684] [General]
全程使用 Docker 为本程序开启HTTPS不在需要在config.json配置需要绑定的公网域名参数了吗？之前版本记得是需要配置两个参数值为外网域名来着

**Comment by limboys** [comment]
![image](https://user-images.githubusercontent.com/36941711/233080402-87c427a5-76bb-4d66-adb2-2192e98a27bc.png)
这一步跳转上方没有在介绍配置公网域名参数了

**Comment by Keldos-Li** [comment]
我记得原先使用docker开启https也没有写config.json的部分吧……

这是我之前编辑的[diff](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/使用教程/_compare/523593b458a8d64f14db26156ed1ce2bcf5dab26...898b07010a3bf333fd64db2d58bf306ecdaaf648)。
不过那些参数或许是要有的，我又在wiki里加回去了。但我也不用docker，不能保证正确性。如有问题请在此反馈，感谢！

**Reply by limboys** [reply]
是的，参数是直接指定在docker run启动时，thanks，我试试是否可以使用
